app/herbivore/herbivore.py

```diff
@@ -0,0 +1,9 @@
+from ..animal.animal import Animal
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        if self.hidden:
+            self.hidden = False
+        else:
+            self.hidden = True
```

Comment:
it is unnecessary to create this module, place all the code in the module `app/main.py`